src/aics-image-viewer/components/App/index.tsx

```diff
@@ -170,14 +171,9 @@ const App: React.FC<AppProps> = (props) => {
   const [image, setImage] = useState<Volume | null>(null);
   const imageUrlRef = useRef<string>("");
 
-  const getNumberOfSlices = (): PerAxis<number> => {
-    if (image) {
-      const { x, y, z } = image.imageInfo.volumeSize;
-      return { x, y, z };
-    }
-    return { x: 0, y: 0, z: 0 };
-  };
-  const numberOfTimesteps = image?.imageInfo.times || 1;
+  const numSlices: PerAxis<number> = image?.imageInfo.volumeSize ?? { x: 0, y: 0, z: 0 };
+  const numSlicesLoaded: PerAxis<number> = image?.imageInfo.subregionSize ?? { x: 0, y: 0, z: 0 };
```

Comment:
in terms of Z slices, is this always either 1 or the total Z ?

Reply:
Currently yeah, this should in practice x and y should always be equal to `volumeSize`, and z should match `volumeSize` in 3d and be 1 in 2d